docs/articles_en/learn_openvino/openvino_samples/get_started_demos.md

```diff
@@ -7,293 +7,330 @@
+            
+               .. code-block:: sh
+               
+                  mkdir build
+               
+               .. note:: 
+               
+                  If you ran the Image Classification verification script during the installation, the C++ samples build directory was already created in your home directory: ``~/openvino_cpp_samples_build/``
+               
+            2. Go to the created directory:
+            
+               .. code-block:: sh
+               
+                  cd build
+            
+            3. Run CMake to generate the Make files for release or debug configuration. For example, for C++ samples:
+            
+               - For release configuration:
+            
+                 .. code-block:: sh
+            
+                    cmake -DCMAKE_BUILD_TYPE=Release <INSTALL_DIR>/samples/cpp
+               
+               - For debug configuration:
+            
+                 .. code-block:: sh
+            
+                    cmake -DCMAKE_BUILD_TYPE=Debug <INSTALL_DIR>/samples/cpp
+               
+            4. Run ``make`` to build the samples:
+            
+               .. code-block:: sh
+               
+                  make
+            
+            For the release configuration, the sample application binaries are in ``<path_to_build_directory>/<architecture>/Release/``; for the debug configuration — in ``<path_to_build_directory>/<architecture>/Debug/``.
+      
+
+.. _select-sample:
+
+Sample Application Setup
+================================
+
+Before running compiled binary files, run the ``setupvars`` script to set all necessary environment variables:
 
-            ################|| Post-processing ||################
+.. tab-set::
 
-            ========== Replacing text in C:\Users\username\Documents\models\public\googlenet-v1\googlenet-v1.prototxt
+   .. tab-item:: Linux
+      :sync: linux
 
-      .. tab-item:: Linux
-         :sync: linux
+      .. code-block:: sh
+   
+         source <INSTALL_DIR>/setupvars.sh
+      
+      The OpenVINO environment variables are removed when you close the shell. As an option, you can permanently set the environment variables as follows:
 
+      1. Open the ``.bashrc`` file in ``<user_home_directory>``:
+      
          .. code-block:: sh
-
-            ###############|| Downloading models ||###############
-
-            ========= Downloading /home/username/models/public/googlenet-v1/googlenet-v1.prototxt
-
-            ========= Downloading /home/username/models/public/googlenet-v1/googlenet-v1.caffemodel
-            ... 100%, 4834 KB, 3157 KB/s, 1 seconds passed
-
-            ###############|| Post processing ||###############
-
-            ========= Replacing text in /home/username/models/public/googlenet-v1/googlenet-v1.prototxt =========
-
-      .. tab-item:: macOS
-         :sync: macos
-
+            
+            vi <user_home_directory>/.bashrc
```

Comment:
I suppose this part is connected to OpenVINO installation and we already recommended users to perform this step in this document https://docs.openvino.ai/2023.2/openvino_docs_install_guides_installing_openvino_from_archive_linux.html#step-2-configure-the-environment in the **Note** section. So, no needs to duplicate the material

Reply:
Applied required changes in the https://github.com/openvinotoolkit/openvino/pull/20957/commits/629ffb13ccb5cf5018dd0385c03a6a064bcb134a commit.